common/lib/protocol-definitions/src/sockets.ts

```diff
@@ -59,6 +59,12 @@ export interface IConnect {
      * If we have standardized features across all services, they need to be exposed in more structured way.
      */
     supportedFeatures?: Record<string, any>;
+
+    /**
+     * Properties that client can send to server to tell info about client environment like loader, driver versions etc.
+     * Format: "prop1:val1;prop2:val2;prop3:val3"
```

Comment:
This comment is more about how we will use it on the client (i.e. references to loader, driver, etc.). But I think it should talk more about expectations in terms of protocol, and such concepts like loader do not exists on this layer. So it should say what client can do (specify a bunch of text separated by ;) and what server can/should do (can log it to better understand usage)

Reply:
Same below